bindgen-integration/src/lib.rs

```diff
@@ -3,6 +3,29 @@ mod bindings {
 }
 
 use std::ffi::CStr;
+use std::os::raw::c_int;
+
+#[test]
+fn test_static_array() {
+    let mut test = unsafe { bindings::Test_COUNTDOWN.as_ptr() };
+    let expected = unsafe { bindings::Test_countdown()};
+    let also_expected = unsafe { bindings::Test_COUNTDOWN_PTR };
+    assert!(!test.is_null());
+    assert_eq!(also_expected, expected, "Pointer value was bogus");
+    assert_eq!(test, also_expected, "Array value was bogus");
```

Comment:
Nitpick: Assertion diagnostic messages should describe expectations, not explain what happens when things go wrong. Expectations read fine whether the assertion is passing (ie when I happen to be reading this code) or when it is printed out because of assertion failure. What is written here only reads OK if the assertion is failing.

Here is my suggested wording:

> bindings::Test_COUNTDOWN.as_ptr() should be the same as bindings::Test_countdown()

but in this particular case, I think the default assert_eq message should be fine.